<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="note_item_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="note_item">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="6">
          <widget class="QPushButton" name="delete_btn">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Base styling for buttons */
QPushButton {
    border: none; /* Remove border for a flat design */
    max-width: 20px; /* Limit button width for a uniform look */
    max-height: 20px; /* Limit button height for a uniform look */
    /* Note: 'icon' property should be set programmatically in PyQt */
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/task_item_icons/icon/task_item/delete_black_48dp.svg</normaloff>:/task_item_icons/icon/task_item/delete_black_48dp.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="7">
          <widget class="QTextEdit" name="note_summary"/>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="edit_btn">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">/* Base styling for buttons */
QPushButton {
    border: none; /* Remove border for a flat design */
    max-width: 20px; /* Limit button width for a uniform look */
    max-height: 20px; /* Limit button height for a uniform look */
    /* Note: 'icon' property should be set programmatically in PyQt */
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/task_item_icons/icon/task_item/edit_black_48dp.svg</normaloff>:/task_item_icons/icon/task_item/edit_black_48dp.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
